<launch>
  <param name="pub_rate" value="100.0"/>
  
  <!-- Launch MoveIt! -->
  <include file="$(find abb_irb2400_moveit_config)/launch/moveit_planning_execution.launch"/>

  <!-- Launch jog controllers -->
  <rosparam command="load"
	    file="$(find jog_controller)/config/abb_irb2400_jog.yaml"/>
  <node name="jog_joint_node" pkg="jog_controller" type="jog_joint_node">
    <param name="use_action" value="true"/>
    <param name="time_from_start" value="1.0"/>
  </node>
  <node name="jog_frame_node" pkg="jog_controller" type="jog_frame_node">
    <param name="use_action" value="true"/>
    <param name="time_from_start" value="1.0"/>
  </node>

</launch>
